lib/oaskit/validation/request_validator.ex

```diff
@@ -87,22 +89,34 @@ defmodule Oaskit.Validation.RequestValidator do
   end
 
   defp validate_parameters(req_data, by_location, jsv_root) do
-    %{path_params: raw_path_params, query_params: raw_query_params} = req_data
-    %{path: path_specs, query: query_specs} = by_location
+    %{path_params: raw_path_params, query_params: raw_query_params, req_headers: req_headers} =
+      req_data
+
+    %{path: path_specs, query: query_specs, header: header_specs} = by_location
+
+    raw_header_params = Map.new(req_headers)
```

Comment:
With `Map.new/1` we will use the value of the last header in case headers are repeated. I feel like we should instead keep the first header value for each key, wdyt?